dagobah/core/core.py

```diff
@@ -117,12 +117,16 @@ def _add_job_from_spec(self, job_json, use_job_id=True):
             job.schedule(job_json['cron_schedule'])
 
         for task in job_json.get('tasks', []):
-            self.add_task_to_job(job,
-                                 str(task['command']),
-                                 str(task['name']),
-                                 soft_timeout=task.get('soft_timeout', 0),
-                                 hard_timeout=task.get('hard_timeout', 0),
-                                 hostname=task.get('hostname', None))
+            # If it is a jobtask, it will have a job_id
+            if task.get('job_id'):
```

Comment:
I think that check is fine. I'm a little more interested in the consistency aspect of building up the Dagobah instance and potentially doing JSON imports. Previously, we could loop over jobs and then process each job's tasks sequentially because they have no other references. Now, we probably need to make sure we do these import-like tasks with respect to the implicit DAG created by jobs referencing each other via jobtasks. Or maybe it doesn't actually matter in practice. Thoughts?